lib/utils.js

```diff
@@ -87,7 +87,7 @@ module.exports.getUnitType = function(srs) {
 
 module.exports.getSpatialResolutions = function() {
   var circumference = 40075000;
-  var zoomLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+  var zoomLevels = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20];
```

Comment:
@GretaCB @isiyu maxzoom is `zoomLevels.length` (https://github.com/mapbox/mapnik-omnivore/blob/moremaxzoom/lib/raster.js#L179), which means that this _is_ 21. That said, previous `maxzoom` was `20` (not 19, as I think we've assumed / verified on master and on branch setup from modularization https://github.com/mapbox/mapnik-omnivore/commit/a0c4cdaec2181dad494c82ca4c5f91ebfb44314b), so I think bumping it to `21` (maxzoom of `22` - 2 more zooms) would be 👍